### DISCUSSION TITLE
We actually work with files now. What else can be done?

### DISCUSSION THREAD

**Discussion by Dum4G** [iptv-org/community#234] [Ideas]
Now, that we actually download playlists from sources there are two additional checks that can be performed:

1. We can get the list of all chunklists from HLS link and remove fixed resolution links from the repo. This automated kind of cleaning must be performed once.
2. We can check if the source is returning simillar chunk by checking for names/sizes/hashes of returning .ts files twice. Helps to detect VODs, broken streams, paid subscription sources, not 24/7 channels

**Comment by freearhey** [comment]
1. Since the script will only need to be run once, maybe you can run it locally and then send the result via PR?

**Reply by Dum4G** [reply]
In case someone make such a script ;)
